pkg/apiserver/logsearch/service.go

```diff
@@ -154,15 +159,8 @@ func (s *Service) GetAllTaskGroups(c *gin.Context) {
 	}
 	var resp = make([]TaskGroupResponse, 0, len(taskGroups))
```

Comment:
This function can return `// @Success 200 {array} TaskGroupModel` directly and no need to loop again building another structure.